src/main/twirl/uk/gov/hmrc/hmrcfrontend/views/pages/HmrcYesNoQuestionPage.scala.html

```diff
@@ -0,0 +1,78 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcStandardPage
+
+@this(
+  hmrcStandardPage: HmrcStandardPage,
+  formWithCSRF: FormWithCSRF,
+  govukErrorSummary: GovukErrorSummary,
+  govukRadios: GovukRadios,
+  govukButton: GovukButton
+)
+
+@(questionMessage: String, form: Form[_], action: Call, pageTitle: Option[String] = None, name: Option[String] = None, backLink: Option[BackLink] = None, submitMessage: Option[String] = None)(implicit request: Request[_], messages: Messages)
```

Comment:
even for a simple Yes/No question page, there's a fair bit that the devs would have to provide, or might want to override - this is going to get complicated really quickly for more complex pages like checkboxes

Reply:
the only bit from me is that I reckon it would be good to move away from having lots of arguments for our layouts to using the case class that encapsulates all params for a template approach, seems like it makes having alternate constructors / helpers for populating stuff easier